dependencies.yaml

```
@@ -172,9 +196,34 @@ dependencies:
     common:
       - output_types: [conda, requirements]
         packages:
-          - *cudf_ver
           - geopandas>=0.11.0
+      - output_types: conda
+        packages:
+          - *cudf_conda
           - rmm=23.04
+    specific:
+      - output_types: requirements
+        matrices:
+          - matrix:
+              cuda: "11.8"
+            packages:
+              - "--extra-index-url=https://pypi.nvidia.com"
+              - rmm-cu11==23.4.*
+          - matrix:
+              cuda: "11.5"
+            packages:
+              - "--extra-index-url=https://pypi.nvidia.com"
+              - rmm-cu11==23.4.*
+          - matrix:
+              cuda: "11.4"
+            packages:
+              - "--extra-index-url=https://pypi.nvidia.com"
+              - rmm-cu11==23.4.*
+          - matrix:
+              cuda: "11.2"
+            packages:
+              - "--extra-index-url=https://pypi.nvidia.com"
+              - rmm-cu11==23.4.*
```

```suggestion
              - "--extra-index-url=https://pypi.nvidia.com"
              - rmm-cu11==23.4.*
          - matrix:
              cuda: "11.5"
            packages:
              - "--extra-index-url=https://pypi.nvidia.com"
              - rmm-cu11==23.4.*
          - matrix:
              cuda: "11.4"
            packages:
              - "--extra-index-url=https://pypi.nvidia.com"
              - rmm-cu11==23.4.*
          - matrix:
              cuda: "11.2"
            packages:
              - "--extra-index-url=https://pypi.nvidia.com"
              - rmm-cu11==23.4.*
```

Can you add some entries to `update-version.sh` to keep this `rmm-cu` packages up-to-date?

You can view `cudf`'s implementation of this below for reference.

- https://github.com/rapidsai/cudf/blob/a7e50920f35c667d041678f6a82d975355cdfd8f/ci/release/update-version.sh#L90-L98